wpiformat/wpiformat/__init__.py

```diff
@@ -246,6 +292,28 @@ def main():
         help=
         "version suffix for clang-format (invokes \"clang-format-CLANG_VERSION\" or \"clang-format\" if no suffix provided)"
     )
+    parser.add_argument(
+        "-tidy-changed",
+        dest="tidy_changed",
+        action="store_true",
+        help=
+        "also runs clang-tidy-CLANG_VERSION on changed files; this requires a compile_commands.json file"
+    )
+    parser.add_argument(
+        "-tidy-all",
+        dest="tidy_all",
+        action="store_true",
+        help=
+        "also runs clang-tidy-CLANG_VERSION on all files (this takes a while); this requires a compile_commands.json file"
+    )
```

Comment:
Can we make this use an optional mutually exclusive group? https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
Done